src/Roastery/Roastery.csproj

```diff
@@ -4,9 +4,10 @@
         <TargetFramework>net8.0</TargetFramework>
         <Nullable>enable</Nullable>
     </PropertyGroup>
-    
+
     <ItemGroup>
         <PackageReference Include="Serilog" Version="3.1.1" />
+        <PackageReference Include="SerilogTracing" Version="1.0.0-dev-00088" />
```

Comment:
The prerelease dependency may be a problem here, we're not expecting SerilogTracing to RTM ahead of this, and having a prerelease dependency will block our .NET tool package publishing.